spack/package.py

```diff
@@ -127,6 +128,7 @@ class Octopus(AutotoolsPackage, CudaPackage):
     depends_on("libyaml", when="+libyaml")
     depends_on("pnfft", when="+pnfft")
     depends_on("nlopt", when="+nlopt")
+    depends_on("dftbplus~mpi", when="+dftbplus")
```

Comment:
we need it as a shared lib and we also need to enable api(dont quite remember why the api is needed but thats the way it is in easybuild at least)
```suggestion
    depends_on("dftbplus~mpi+api+sharedlibs~openmp", when="+dftbplus")
```

Reply:
None of `api`, `sharedlibs`, `openmp` exist as variants. Is this as different package? Or a significant update?